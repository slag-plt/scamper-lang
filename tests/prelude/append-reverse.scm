(append (list "ab" "ba") (list 1 2 3) (list))
(append (list) (list))
(append (list "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld")
 (list "Hello" "World"))
(append (list #t #t #t #t) (list #f #f #f #f) (list "a" "a"))
(append (list 10 10 10 10 10) (list #t #t) (list "abab" "abab"))
(reverse (list "ab" "ba"))
(reverse (list))
(reverse (list "Hello" "World"))
(reverse (list "abab" "abab"))
(reverse (list 1 2 3))