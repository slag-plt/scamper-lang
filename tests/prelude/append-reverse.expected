(list "ab" "ba" 1 2 3)
null
(list "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "Hello" "World")
(list #t #t #t #t #f #f #f #f "a" "a")
(list 10 10 10 10 10 #t #t "abab" "abab")
(list "ba" "ab")
null
(list "World" "Hello")
(list "abab" "abab")
(list 3 2 1)
